ecosystem/nft-metadata-crawler/nft-metadata-crawler-parser/src/models/nft_metadata_crawler_uris_query.rs

```diff
@@ -0,0 +1,64 @@
+// Copyright © Aptos Foundation
+
+use crate::schema::nft_metadata_crawler_uris;
+use diesel::{
+    prelude::*,
+    r2d2::{ConnectionManager, PooledConnection},
+};
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Deserialize, Identifiable, Queryable, Serialize)]
+#[diesel(primary_key(token_uri))]
+#[diesel(table_name = nft_metadata_crawler_uris)]
+pub struct NFTMetadataCrawlerURIsQuery {
+    pub token_uri: String,
+    pub raw_image_uri: Option<String>,
+    pub raw_animation_uri: Option<String>,
+    pub cdn_json_uri: Option<String>,
+    pub cdn_image_uri: Option<String>,
+    pub cdn_animation_uri: Option<String>,
+    pub json_parser_retry_count: i32,
+    pub image_optimizer_retry_count: i32,
+    pub animation_optimizer_retry_count: i32,
+    pub last_updated: chrono::NaiveDateTime,
+}
+
+impl NFTMetadataCrawlerURIsQuery {
+    pub fn get_by_token_uri(
```

Comment:
I dont' see callsites here so are we doing the retry thing? I would just in case there's a temporary connection error or something.